src/components/UI/dropdown/dropdown.module.scss

```diff
@@ -0,0 +1,143 @@
+// scss variables
+$border-color: #000000;
+$black: #000000;
+$text-color: #ffffff;
+$hover-color: #49a82e;
+$background-color: #ffffff;
+
+.userGreet {
+  color: $text-color;
+  margin: 23px;
+}
+
+.userGreet a {
+  padding: 0px;
+}
+
+.userProfilePic {
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+  display: inline;
+  vertical-align: middle;
+}
+.dropdownHeader {
+  margin: 10px;
+  cursor: pointer;
+}
+
+.userGreetMsg {
+  display: inline;
+  vertical-align: middle;
+  margin-right: 10px;
+  color: rgb(255, 255, 255);
+}
+
+.userGreetMsg:hover {
+  color: $hover-color;
+}
+
+.myProfileWrapper {
+  padding-bottom: 15px;
+  text-align: center;
+  border-bottom: 1px solid $border-color;
+}
+
+.myProfile:hover {
+  color: $hover-color;
+}
+
+.myProfile {
+  color: $black;
+  text-decoration: none;
+}
+
+.signOutWrapper {
+  padding-top: 15px;
+  text-align: center;
+  border-top: 1px solid $border-color;
+}
+
+.signOutWrapper > button {
+  all: unset;
+  cursor: pointer;
+}
+
+.signOutWrapper button:focus-visible {
+  outline: var(--black) 5px auto;
```

Comment:
why are you using `var(--black)` here, I think this is not valid

Reply:
okay I will make a changed to it

Reply:
okay thanks I will changed that.